Google Play Music/CoreMusicApp.cs

```diff
@@ -176,6 +176,11 @@ private void nextTrack()
             GPMBrowser.EvaluateScriptAsync("(function() {document.querySelectorAll('[data-id=forward]')[0].click()})()");
         }
 
+        private void findFocus()
+        {
+            GPMBrowser.EvaluateScriptAsync("(function() {document.getElementById('input').focus()})()");
```

Comment:
Just being picky now but could we update this selector to allow for Google's tendency to change ID's and classes of elements.  Something like `document.querySelectorAll('sj-search-box > input')[0].focus()` should be future proof enough I think.